<robot name="myrobot">

<link name="base_link">
<visual>
	<origin xyz="0 0 0.005" rpy="0 0 0"/>
		<geometry>
		<box size="0.01 0.01 0.01"/>
	</geometry>
</visual>	
</link>
<link name="part1">
<visual>
  <origin xyz="0 0 0.07875" rpy="0 0 0"/>
    <geometry>
      <cylinder length="0.1575" radius="0.030"/>
    </geometry>
</visual>
</link>
<link name="part2">
<visual>
  <origin xyz="0 0 0.09167" rpy="0 0 0"/>
    <geometry>
      <cylinder length="0.18334" radius="0.025"/>
    </geometry>
</visual>
</link>


<joint name="joint1" type="continuous">
    <parent link="base_link"/>
    <child link="part1"/>
    <origin xyz="0 0 0.0" rpy="0 0 0" /> 
    <axis xyz="0 1 0" rpy="0 0 0" />
  </joint>
<joint name="joint2" type="continuous">
    <parent link="part1"/>
    <child link="part2"/>
    <origin xyz="0 0 0.1575" rpy="0 0 0" /> 
    <axis xyz="0 1 0" rpy="0 0 0" />
</joint>
</robot>
